ci/default.yml

```
@@ -7,11 +7,11 @@ include:
     - nox -s test_model_stencils-3.10 -- --backend=$BACKEND --grid=$GRID
   parallel:
     matrix:
-    - BACKEND: [gtfn_cpu, gtfn_gpu, roundtrip]
+    - BACKEND: [gtfn_cpu, gtfn_gpu, embedded]
```

Minor thing: we can put `embedded` first in the list (alphabetical order + default backend)
```suggestion
    - BACKEND: [embedded, gtfn_cpu, gtfn_gpu]
```